entries/c/chatwork.com.json

```
@@ -0,0 +1,14 @@
+{
+  "Chatwork": {
+    "domain": "chatwork.com",
+    "url": "https://go.chatwork.com/",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://support-en.chatwork.com/hc/en-us/articles/360023097632",
```

```suggestion
    "documentation": "https://support-en.chatwork.com/hc/en-us/articles/360023097632",
    "notes": "SMS 2FA is only available to users on paid plans",
```

According to [this page](https://support-en.chatwork.com/hc/en-us/articles/360023097452):

> * Two-factor authentication using SMS cannot be used with the free plan. Please use two-factor authentication using the authentication app.

---

I overlooked. Thank you 👍